lib/model-interfaces/langchain/functions/request-handler/adapters/bedrock/claude.py

```diff
@@ -2,7 +2,7 @@
 
 import boto3
 from langchain.agents import ZeroShotAgent
-from langchain.llms import Bedrock
+from langchain.chat_models import Bedrock
```

Comment:
Not sure about this line...

Looks like [chat_models](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/chat_models/__init__.py#L24) exposes only `BedrockChat` and not `Bedrock` ?

Reply:
Good catch! It should import from `llms` and not `chat_models`